### ISSUE TITLE
Tap event is not registered on iOS when clicking the checkbox

### ISSUE THREAD

**Issue by bnussey** [bradmartin/nativescript-checkbox#3]
In iOS when you click on the checkbox, the tap event is not fired, however the tap event fires  when clicking on the text.

**Comment by bnussey** [comment]
Looks like this is still an issue...

**Comment by sitefinitysteve** [comment]
Sorry, i've just been craaaazy busy... PR submitted, just needed to send the notify

You can just edit your checkbox.ios.js right now and find the "BEMCheckBoxDelegateImpl.prototype.didTapCheckBox = function (checkBox) {" function, change it to this

```
BEMCheckBoxDelegateImpl.prototype.didTapCheckBox = function (checkBox) {
        var owner = this._owner.get();
        if (owner) {
            var eventData = {
                eventName: "tap",
                object: owner
            };
            
            owner.notify(eventData);

            owner._onPropertyChangedFromNative(CheckBox.checkedProperty, checkBox.on);
        }
    };
```

**Comment by bradmartin** [comment]
Published 1.1.5
